docs/entities/Dev_Install.txt

```diff
@@ -0,0 +1,249 @@
+Full Install of BC-ROS Development Environment - Windows version
+================================================================
+
+BASIC WINDOWS INSTALL
+---------------------
+1. fresh Windows install (Windows 10 Pro build 2004 or later)
+  - 8+ GB RAM
+  - 60+ GB disk
+  - if using a VM, virtualize Intel VT-x/EPT or AMD-V/RI (also needed for Docker later)
+  - set power plan to Performance
+  - un-pin default apps
+  - show all taskbar icons
+  - show This PC on desktop
+  - uninstall unwanted Windows apps (Cortana, OneDrive, Teams, Skype, etc)
+2. install Avira anti-virus or equivalent
+3. install Chrome + extensions
+  - Avira Browser Safety
+  - Auto Tab Discard
+  - LastPass
+  - set as default browser
+4. install Notepad++ (optional)
+5. install Acrobat Reader
+6. install Postman (for API testing)
+7. install pgAdmin (for db access)
+8. install WinMerge (optional)
+9. install PuTTY Authentication Agent (Pagent) to use SSH for GH (optional)
+  - copy id_rsa* + *.ppk to ~/.ssh
+  - add key
+  - add Pagent shortcut to Startup folder
+12. install Office 365 (optional)
+  - configure Outlook
+13. install Firefox (optional)
+14. install BC Sans fonts (Google for zip file, install the TTFs)
+15. install Zoom (from Microsoft Store)
+16. install Spotify (from Microsoft Store)
+17. install Teams (from Microsoft Store)
+18. install vim or nvim (and know how to exit vim) 
+
+QUICK DEV ENVIRONMENT SETUP FOR FRONTEND DEVELOPMENT
+----------------------------------------------------
+
+- Fork all the 3 frontend UIs: Filings UI, Create UI, and Edit UI. 
+
+- Clone the 3 UIs on your machine. For reference: https://github.com/argush3/entity/blob/master/docs/setup-forking-workflow.md 
+
+- Download Node Version 16.15.0: https://nodejs.org/dist/v16.15.0/node-v16.15.0-x64.msi 

```

Comment:
Same comment as the other one, the version. I think any version of (`20.x.x`) should work as Sev uses another version of 20 as mine.